PH-1/app/ingestion/router.py

```
@@ -44,4 +46,31 @@ def validate_file_strict(file: UploadFile) -> None:
+        image_bytes = file.file.read()
+        image = Image.open(io.BytesIO(image_bytes))
+        if image.mode != 'RGB':
+            image = image.convert('RGB')
+        text = pytesseract.image_to_string(image, lang='eng')
+        return text.strip() if text.strip() else None
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"OCR failed: {str(e)}")
+
+        
+router = APIRouter()
+@router.post("/upload-image")
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)